src/monitor.cpp

```diff
@@ -335,13 +335,12 @@ String getPoolAPIUrl(void) {
         else {
             switch (Settings.PoolPort) {
                 case 3333:
-                    if (Settings.PoolAddress == "pool.vkbit.com")
-                        poolAPIUrl = "https://vkbit.com/miner/";
-                    else if (Settings.PoolAddress == "pool.sethforprivacy.com")
+                    if (Settings.PoolAddress == "pool.sethforprivacy.com")
```

Comment:
this change doesn't seem to be related with touchhandler and 1bit fix

Reply:
yup. hijacked pr to correct my unreleased code after reading https://github.com/BitMaker-hub/NerdMiner_v2/pull/410  not clear on the reasoning,  it did  function at the time of testing.
This adjustment is mentioned in this pr summary along with OneButton.